zerver/lib/topic.py

```diff
@@ -160,7 +160,9 @@ def update_messages_for_topic_edit(
     if propagate_mode == "change_later":
         propagate_query = propagate_query & Q(id__gt=edited_message.id)
 
-    messages = Message.objects.filter(propagate_query).select_related()
+    messages = Message.objects.filter(propagate_query).select_related(
+        "sender", "sender__realm", "realm", "sending_client", "recipient"
```

Comment:
I think we want to extract this list as a variable like `Message.DEFAULT_SELECT_RELATED` rather than having it appear in some many places.